### PR TITLE
Support Rails 6

### PR DESCRIPTION
**Pull request by chamnap**
Specs are running fine with rails 6.

### CODE REVIEW

File: starburst.gemspec [hunk 1]
```diff
@@ -16,9 +16,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
 
-  s.add_dependency "rails", ">= 4.2.0", "< 6.0"
-
-  s.add_development_dependency "sqlite3", "~> 1.3.9"
```
Comment (erichmachado) [inline starburst.gemspec]: We need to keep this here to support local development without `Appraisal`. Moving the `sqlite3` line together with the other development dependencies would be good though.

File: gemfiles/rails_6.0.0.gemfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails",   "~> 6.0.0"
+gem "sqlite3", "~> 1.4.1"
```
Comment (erichmachado) [inline gemfiles/rails_6.0.0.gemfile]: I understand you're doing this to fix the issue that Rails 6 requires `sqlite` version `>= 1.4`. A better approach would be to keep the development dependency set on the `gemspec` and only override the version where needed directly on the `Appraisals` file.

Supposing the `sqlite` version is kept at the current version on the `gemspec`, we could do this on the `Appraisals` for example:

```ruby
appraise 'rails_6.0.0' do
  gem 'rails', '~> 6.0.0'
  # Rails 6 requires a newer version of the sqlite3 gem
  gem 'sqlite3', '~> 1.4.0'
end

%w[5.2.0 5.1.0 5.0.0 4.2.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
  end
end
```
Another approach would be to update the `sqlite3` version on the `gemspec` to `~> 1.4.0` and then have this `Appraisals` file instead:
```ruby
%w[6.0.0 5.2.0 5.1.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
  end
end

%w[5.0.0 4.2.0].each do |rails_version|
  appraise "rails_#{rails_version}" do
    gem 'rails', "~> #{rails_version}"
    gem 'sqlite3', "~> 1.3.0"
  end
end
```
The latter might be better for long-term maintainability, as the tendency is to remove support for older versions of the framework over time as they become deprecated.

You'll need to update the `appraisal` files after that with:
```sh
$ appraisal clean
$ appraisal generate
```

Comment (chamnap) [reply gemfiles/rails_6.0.0.gemfile]: Thanks for the tips!

**Comment by chamnap** [comment]
@erichmachado, I have changed based on your comments.

**Comment by erichmachado** [comment]
I'm merging this @csm123. I think it would be good for us to plan a new gem release after that.

**Pull request merged**